rs/https_outcalls/service/proto/https_outcalls_service/v1/proto.proto

```diff
@@ -21,6 +21,7 @@ message HttpsOutcallRequest {
   HttpMethod method = 4;
   uint64 max_response_size_bytes = 5;
   bool socks_proxy_allowed = 6;
```

Comment:
maybe add a comment that we plan to remove the bool socks_proxy_allowed in favour of socks_proxy_addrs

Reply:
done